From 617741a81a502b857604ed8fc4f44b528b05b431 Mon Sep 17 00:00:00 2001
From: Xiao Yindong <xydarcher@uestc.edu.cn>
Date: Tue, 24 Oct 2023 01:28:57 +0800
Subject: [PATCH] fix runtime error on msys2.

---
 misc/launcher.c | 3 ---
 1 file changed, 3 deletions(-)

diff --git a/misc/launcher.c b/misc/launcher.c
index 49d6414e7..afdff8ec3 100644
--- a/misc/launcher.c
+++ b/misc/launcher.c
@@ -146,9 +146,6 @@ char *find_exe(char *exename, char *script) {
         easy_install-ed scripts use the absolute path to python[w].exe
     */
     _splitpath(script, drive, dir, fname, ext);
-    result = dir + strlen(dir) -1;
-    if (*result == '\\') result--;
-    while (*result != '\\' && result>=dir) *result-- = 0;
     _makepath(path, drive, dir, exename, NULL);
     return loadable_exe(path);
 }
-- 
2.38.1.windows.1

